src/types/Patterns.ts

```diff
@@ -18,9 +18,30 @@
  * SPDX-License-Identifier: Apache-2.0
  ********************************************************************************/
 
+import { getRequireHttpsUrlPattern } from 'services/EnvironmentService'
+
+// check the REQUIRE_HTTPS_URL_PATTERN environment variable, defaulting to 'true' if not set
+const requireHttpsUrlPattern = getRequireHttpsUrlPattern() === 'true'
+if (requireHttpsUrlPattern) {
+  console.log('HTTPS in URL patterns is required.')
+} else {
+  console.log('HTTPS in URL patterns is not required.')
+}
```

Comment:
console.logs L25-29 should be removed. Info is not relevant enough to be printed every time. It's easy to check the value in index.html